packages/user-interface/src/components/LinksList.tsx

```diff
@@ -33,6 +39,16 @@ const LinksList = ({
     );
   }
 
+  if (!links || links.length === 0) {
+    if (!showEmpty) return null;
+    return (
+      <section>
+        <SectionHeader title={title} />
+        <Paragraph>{emptyMessage}</Paragraph>
+      </section>
+    );
+  }
+
```

Comment:
Dit voelt een beetje onnodig aan. Wanneer zou je die linklist willen tonen als die leeg is?

Reply:
Ja nog geen situatie voor op dit moment. Haal het voor nu wel ff weg dan.